### ISSUE TITLE
Laravel 8 support

### ISSUE THREAD

**Issue by patie** [beyondcode/helo-laravel#40]
Hi, 

im trying install `composer require --dev beyondcode/helo-laravel`

Versions from composer.lock
```
laravel/framework 8.83.27
swiftmailer/swiftmailer 6.3.0
```

Error
```
[2023-03-21 14:19:56] local.ERROR: Class 'BeyondCode\HeloLaravel\Swift_Mailer' not found {"exception":"[object] (Error(code: 0): Class 'BeyondCode\\HeloLaravel\\Swift_Mailer' not found at /app/vendor/beyondcode/helo-laravel/src/CreatesMailers.php:41)
[stacktrace]
#0 /app/vendor/beyondcode/helo-laravel/src/HeloLaravelServiceProvider.php(56): BeyondCode\\HeloLaravel\\HeloLaravelServiceProvider->createLaravel7Mailer(Object(Illuminate\\Foundation\\Application))
#1 /app/vendor/laravel/framework/src/Illuminate/Container/Container.php(873): BeyondCode\\HeloLaravel\\HeloLaravelServiceProvider->BeyondCode\\HeloLaravel\\{closure}(Object(Illuminate\\Foundation\\Application), Array)
#2 /app/vendor/laravel/framework/src/Illuminate/Container/Container.php(758): Illuminate\\Container\\Container->build(Object(Closure))
#3 /app/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(851): Illuminate\\Container\\Container->resolve('BeyondCode\\\\Helo...', Array, true)
#4 /app/vendor/laravel/framework/src/Illuminate/Container/Container.php(694): Illuminate\\Foundation\\Application->resolve('BeyondCode\\\\Helo...', Array)
#5 /app/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(836): Illuminate\\Container\\Container->make('BeyondCode\\\\Helo...', Array)
#6 /app/vendor/beyondcode/helo-laravel/src/HeloLaravelServiceProvider.php(66): Illuminate\\Foundation\\Application->make('BeyondCode\\\\Helo...')
#7 /app/vendor/beyondcode/helo-laravel/src/HeloLaravelServiceProvider.php(25): BeyondCode\\HeloLaravel\\HeloLaravelServiceProvider->bootMailable()
#8 /app/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): BeyondCode\\HeloLaravel\\HeloLaravelServiceProvider->boot()
#9 /app/vendor/laravel/framework/src/Illuminate/Container/Util.php(40): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#10 /app/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#11 /app/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(37): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Illuminate\\Foundation\\Application), Array, Object(Closure))
#12 /app/vendor/laravel/framework/src/Illuminate/Container/Container.php(653): Illuminate\\Container\\BoundMethod::call(Object(Illuminate\\Foundation\\Application), Array, Array, NULL)
#13 /app/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(924): Illuminate\\Container\\Container->call(Array)
#14 /app/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(905): Illuminate\\Foundation\\Application->bootProvider(Object(BeyondCode\\HeloLaravel\\HeloLaravelServiceProvider))
#15 [internal function]: Illuminate\\Foundation\\Application->Illuminate\\Foundation\\{closure}(Object(BeyondCode\\HeloLaravel\\HeloLaravelServiceProvider), 16)
#16 /app/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(906): array_walk(Array, Object(Closure))
#17 /app/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/BootProviders.php(17): Illuminate\\Foundation\\Application->boot()
#18 /app/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(237): Illuminate\\Foundation\\Bootstrap\\BootProviders->bootstrap(Object(Illuminate\\Foundation\\Application))
#19 /app/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(310): Illuminate\\Foundation\\Application->bootstrapWith(Array)
#20 /app/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(127): Illuminate\\Foundation\\Console\\Kernel->bootstrap()
#21 /app/artisan(37): Illuminate\\Foundation\\Console\\Kernel->handle(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#22 {main}
"} 

   Error 

  Class 'BeyondCode\HeloLaravel\Swift_Mailer' not found

  at vendor/beyondcode/helo-laravel/src/CreatesMailers.php:41
     37▕         $defaultDriver = $app['mail.manager']->getDefaultDriver();
     38▕         $config = $this->getConfig($defaultDriver);
     39▕ 
     40▕         // Laravel 7 no longer bindes the swift.mailer:
  ➜  41▕         $swiftMailer = new Swift_Mailer($app['mail.manager']->createTransport($config));
     42▕ 
     43▕         // Once we have create the mailer instance, we will set a container instance
     44▕         // on the mailer. This allows us to resolve mailer classes via containers
     45▕         // for maximum testability on said classes instead of passing Closures.
```

any idea? thank you

**Comment by patie** [comment]
Its look like this pull request is probably related to my problem https://github.com/beyondcode/helo-laravel/pull/13

**Comment by McGo** [comment]
Shoudl be fixed with PR #41
